tfx/components/schema_gen/executor_test.py

```diff
@@ -46,11 +46,48 @@ def testDo(self):
     input_dict = {
         executor.STATISTICS_KEY: [statistics_artifact],
     }
+
+    exec_properties = {
+        'infer_feature_shape': False,
+        'exclude_splits': None
+    }
+
     output_dict = {
         executor.SCHEMA_KEY: [schema_output],
     }
 
-    exec_properties = {'infer_feature_shape': False}
+    schema_gen_executor = executor.Executor()
+    schema_gen_executor.Do(input_dict, output_dict, exec_properties)
+    self.assertNotEqual(0, len(tf.io.gfile.listdir(schema_output.uri)))
+
+  def testDoWithExcludeSplits(self):
```

Comment:
merge test as comments above

and update_schema is not tested, consider create a fake third split in testdata (e.g., `test` that copy from `eval`) thus you can test both infer and update schema

Reply:
i tested update_schema and added log statements here as well.